geonode_qgis_server/signals.py

```diff
@@ -179,7 +179,11 @@ def qgis_server_post_save(instance, sender, **kwargs):
     thumbnail_remote_url += reverse(
         'qgis-server-thumbnail', kwargs={'layername': instance.name})
     logger.debug(thumbnail_remote_url)
-    create_thumbnail(instance, thumbnail_remote_url, ogc_client=http_client)
+
+    from geonode_qgis_server.tasks.update import create_qgis_server_thumbnail
```

Comment:
I tried to avoid cyclic dependency, and also to follow geonode code in other similar tasks in geoserver thumbnail update.